### PR TITLE
tests: suit: Reenable signature checking on native_posix

### PR DESCRIPTION
**Pull request by ahasztag**
Fixed and reanabled signature checking.

### CODE REVIEW

**Review by tomchy — APPROVED** [review]

File: tests/subsys/suit/fetch_integrated_payload_flash/src/manifest_52.c [hunk 1]
```diff
@@ -18,12 +18,12 @@ uint8_t manifest_buf[] = {
...
-	0x18, 0xA0, 0xBB, 0x64, 0x5A, 0x4E, 0x00, 0x03, 0xDE, 0x93, 0x17, 0x43, 0xAD, 0x98, 0x79,
-	0xF1, 0x47, 0xB8, 0x32, 0xB4, 0xC0, 0x6F, 0x8F, 0xAF, 0xE5, 0x81, 0xF2, 0xE3, 0x6B, 0xEC,
-	0x0A, 0x80, 0x59, 0x94, 0xF7, 0x0E, 0xAA, 0x57, 0x8A, 0xDE, 0x84, 0x03, 0xBC, 0x69, 0x23,
-	0x56, 0x60, 0xEF, 0x49, 0x2F, 0xAA, 0x8C, 0x7E, 0x03, 0x58, 0xFD, 0xA8, 0x01, 0x01, 0x02,
+	0x58, 0x51, 0xD2, 0x84, 0x4A, 0xA2, 0x01, 0x26, 0x04, 0x45, 0x1A, 0x40, 0x00, 0x00, 0x00,
+	0xA0, 0xF6, 0x58, 0x40, 0xD4, 0x80, 0xAE, 0xC9, 0x57, 0x17, 0x8D, 0xB0, 0x41, 0x36, 0xDF,
+	0x95, 0x85, 0x8E, 0x40, 0x3D, 0xE1, 0xB5, 0x34, 0xEA, 0x14, 0xDA, 0xBB, 0xC4, 0x8E, 0x8D,
+	0xED, 0x45, 0x08, 0xBD, 0x1A, 0xC4, 0x9E, 0xD6, 0x2C, 0xD1, 0x17, 0xFB, 0x81, 0x22, 0xBB,
+	0xA2, 0xA5, 0xD2, 0xC7, 0x7A, 0x75, 0x8C, 0xA5, 0xE3, 0xA8, 0x59, 0x7D, 0x99, 0xA3, 0xF6,
+	0xF1, 0x76, 0x8C, 0xAB, 0x6E, 0x23, 0xCF, 0x75, 0x03, 0x58, 0xFD, 0xA8, 0x01, 0x01, 0x02,
```
Comment (nordicjm) [inline tests/subsys/suit/fetch_integrated_payload_flash/src/manifest_52.c:26]: I was told binary files should not be in sdk-nrf and should be put elsewhere e.g. nrfxlib @carlescufi

Comment (tomchy) [reply tests/subsys/suit/fetch_integrated_payload_flash/src/manifest_52.c:26]: Does a C file with ASCII characters qualify as a binary file?
Those are test vectors for the interpreter that cannot handle human-readable inputs.

FYI: Without this PR, all dev in SUIT is blocked due to the red CI after latest upmegre.

**Review by kszromek-nordic — APPROVED** [review]

File: tests/subsys/suit/fetch_integrated_payload_flash/src/manifest_52.c (hunk 1 above)
Comment (ahasztag) [reply tests/subsys/suit/fetch_integrated_payload_flash/src/manifest_52.c:26]: This is not something which is really introduced this PR, such a change would require a massive amount of work to rework the way the manifests are tested.

We quickly need this fix, so this is a topic to discuss separately from this PR.

**Review by nordicjm — DISMISSED** [review]
Awaiting comment from @carlescufi

**Review by carlescufi — APPROVED** [review]
Just had the time to look at this. These are tiny, pre-built manifests stored as C arrays in text mode. We have similar "pre-calculated" output tables in many crypto/protocol tests both in NCS and Zephyr, so this is acceptable.

**Pull request merged**